### PR TITLE
Fix misprint in unit test

### PR DESCRIPTION
**Pull request by aprokop**

### CODE REVIEW

File: test/Search_UnitTestHelpers.hpp [hunk 1]
```diff
@@ -186,7 +186,7 @@ void checkResults(ArborX::DistributedSearchTree<DeviceType> const &tree,
   BOOST_TEST(indices_host == indices_ref, tt::per_element());
   BOOST_TEST(offset_host == offset_ref, tt::per_element());
   BOOST_TEST(ranks_host == ranks_ref, tt::per_element());
-  BOOST_TEST(distances_host != distances_ref, tt::per_element());
+  BOOST_TEST(distances_host == distances_ref, tt::per_element());
```
Comment (dalg24) [inline test/Search_UnitTestHelpers.hpp]: How are both the new and the old version passing the tests? :'(

Comment (aprokop) [reply test/Search_UnitTestHelpers.hpp]: I hope they are running... the whole thing is behind `#ifdef ARBORX_ENABLE_MPI`. If that does not work, all bets are off.

Comment (masterleinad) [reply test/Search_UnitTestHelpers.hpp]: What do you see if you just print the values?

**Comment by aprokop** [comment]
Just checked and the sizes of `distances` is always zero. And yes, I remember having that conversation not too long ago that we never check distances...

**Comment by masterleinad** [comment]
Independent of the size of `distances_ref` this change is correct.

**Pull request merged**